### ISSUE TITLE
Option to render web_Assets w/o rendering tiles.

### ISSUE THREAD

**Issue by KariTrace** [overviewer/Minecraft-Overviewer#570]
[10:07] <Kari|work> @ agrif: is there a way for overviewer to only copy and setup the web_assets w/o beginning the actual rendering?
[10:08] <agrif> I don't think so
[10:08] <agrif> that might be a useful thing to include :/
[10:10] <Kari|work> Indeed, I'm working on the web_assets part of things now that I have my render pipeline working. Adding all the fancy stuff, but I have to cancel the render each time :(. should I make a ticket for it?
[10:13] <agrif> probably
[10:13] <agrif> even if noone else agrees with it, it's a good idea to be sure we don't forget to discuss it

TL;DR CLI option to render only web_assets and not (re)render the tiles.
